### PR TITLE
Exclude testJitserverArguments on xlinux jdk8

### PR DESCRIPTION
**Pull request by pshipton**
Issue https://github.com/eclipse-openj9/openj9/issues/18599

### CODE REVIEW

File: test/functional/cmdLineTests/jitserver/playlist.xml [hunk 1]
```diff
@@ -27,6 +27,11 @@
 		<variations>
 			<variation>NoOptions</variation>
 		</variations>
+			<disable>
+				<comment>https://github.com/eclipse-openj9/openj9/issues/18599</comment>
+				<version>8</version>
+				<platform>x86-64_linux</platform>
```
Comment (pshipton) [inline test/functional/cmdLineTests/jitserver/playlist.xml:33]: Do I need to add `.*` to this?

Comment (llxia) [reply test/functional/cmdLineTests/jitserver/playlist.xml]: No, it is not necessary to have `.*` in this case. But we do need to wrap this with `<disables>...</disables>`.

Comment (pshipton) [reply test/functional/cmdLineTests/jitserver/playlist.xml]: Updated.

**Review by llxia — APPROVED** [review]
LGTM

**Comment by llxia** [comment]
Forgot to provide the [Grinder link](https://hyc-runtimes-jenkins.swg-devops.com/view/Test_grinder/job/Grinder/45643/)

**Pull request merged**